src/db/collections/outputs/mod.rs

```diff
@@ -821,25 +821,25 @@ mod analytics {
 
         /// Gets the number of claimed tokens.
         #[tracing::instrument(skip(self), err, level = "trace")]
-        pub async fn get_claimed_token_analytics(
+        pub async fn get_unclaimed_token_analytics(
             &self,
             ledger_index: MilestoneIndex,
-        ) -> Result<ClaimedTokensAnalytics, Error> {
+        ) -> Result<UnclaimedTokensAnalytics, Error> {
             Ok(self
                 .aggregate(
                     vec![
                         doc! { "$match": {
                             "metadata.booked.milestone_index": { "$eq": 0 },
-                            "metadata.spent_metadata.spent.milestone_index": { "$lte": ledger_index },
+                            "metadata.spent_metadata.spent.milestone_index": { "$not": { "$lte": ledger_index } }
```

Comment:
just making sure again: `$not $lte` matches either: `spent.milestone_index > ledger_index` or `spent_metadata` is missing, right?  And we do it this way because it's an optimization over having 3 matches with a `$gt` and an explicit check for existence of `spent_metadata`.

Reply:
Yeah, I think that's it.

Reply:
Yep!